app/src/screens/Home/index.tsx

```diff
@@ -56,6 +57,11 @@ const Home: React.FC = () => {
     if (logout.rejected.match(logoutAction)) {
       console.error(logoutAction.payload || logoutAction.error);
     }
+
+    const unregisterAction = await dispatch(unregister());
```

Comment:
Do we only use this for testing?

Reply:
Not just testing, it will actually fire in real world usage when the user logs out. But that feels "right" (but it is a bit out of scope of this ticket lol).